docs/installation.rst

```diff
@@ -193,6 +193,8 @@ ports.
 After you have installed the dependencies, you can proceed with steps outlined
 under :ref:`pip-installation`.
 
+.. _debian-ubuntu:
+
```

Comment:
no need for this.

Reply:
Would this one be redundant, too? : 

https://github.com/borgbackup/borg/blob/0e711b66728b0c2ff4113eb5df32326a9ce85f9e/docs/installation.rst?plain=1#L23-L26